src/cr/cube/stripe.py

```diff
@@ -558,12 +560,13 @@ def unweighted_count(self):
 class _MeansStripeRow(_BaseStripeRow):
     """Stripe-row for a non-MR 1D cube-result with means."""
 
-    def __init__(self, element, unweighted_count, mean):
```

Comment:
Are we even using this anywhere? Doesn't seem like it. If so - please get rid of it :)

Reply:
mmm i don't understand this comment :D